src/frontend/src/pages/myfeeds.js

```diff
@@ -208,7 +212,9 @@ export default function MyFeeds() {
                           color={submitStatus.isError ? 'error' : 'textPrimary'}
                           align="center"
                         >
-                          {submitStatus.message}
+                          {alert === true ? (
```

Comment:
`{alert ? (...) : null}`

Reply:
I tried this at first. It will render an empty SnackBar component